classes/CategoryHelper.php

```diff
@@ -0,0 +1,97 @@
+<?php
+/**
+ * Contao Open Source CMS
+ *
+ * Copyright (c) 2017 Heimrich & Hannot GmbH
+ *
+ * @author  Rico Kaltofen <r.kaltofen@heimrich-hannot.de>
+ * @license http://www.gnu.org/licences/lgpl-3.0.html LGPL
+ */
+
+namespace NewsCategories;
+
+class CategoryHelper
+{
+
+    /**
+     * Prepare the category object
+     *
+     * @param NewsCategoryModel $objCategory
+     *
+     * @return object
+     */
+    public static function prepareCategory(NewsCategoryModel $objCategory)
+    {
+        $category                = (object) $objCategory->row();
```

Comment:
Object access is more user-friendly than array access especially in template usage. $this->category['name'] vs $this->category->name. You prefer arrays?